cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -3,6 +3,8 @@
 // Required modules.
 var arrayHelpers = require( '../modules/util/array-helpers' );
 var typeCheckers = require( '../modules/util/type-checkers' );
+var domTraverse = require( '../modules/util/dom-traverse' );
+var domManipulators = require( '../modules/util/dom-manipulators' );
```

Comment:
I would make this `domCreate = require( '../modules/util/dom-manipulators' ).create`.